frontend-opal/components/WachList2/index.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import { View, Text, Image } from 'react-native';
+import styles from './styles';
+
+const WatchList2 = () => (
```

Comment:
@martinupset instead of having 2 components lets have one component where you have a minimal logic of:
- clicking on closed line opens the details
- clicking on opened details closes the line

Reply:
ok